src/Ship.js

```diff
@@ -0,0 +1,30 @@
+class Ship {
+  constructor(itinerary){
+    this.itinerary = itinerary;
+    this.currentPort = itinerary.ports[0];
+    this.previousPort = null;
+    this.currentPort.addShip(this);
+  };
```

Comment:
Could a space here for better readability:
```suggestion
  };

```